fcl/caf/cafmaker_common_defs.fcl

```diff
@@ -0,0 +1,16 @@
+#include "eventweight_genie_sbn.fcl"
+#include "eventweight_flux_sbn.fcl"
+
+#include "mcsproducer.fcl"
+#include "rangeproducer.fcl"
+
+# This file if a place to define things which are common
+# between the SBND and ICARUS cafmaker configurations.
+# For now this just means a few producers listed below.
+cafmaker_common_producers: {
+  pandoraTrackMCS:   @local::mcs_sbn
+  pandoraTrackRange:   @local::range_sbn
+  rns: { module_type: "RandomNumberSaver" }
+  genieweight: @local::sbn_genie_eventweight
+  fluxweight: @local::sbn_eventweight_flux  
+}
```

Comment:
Recommending to set this in a `PROLOG`:
```suggestion
}

END_PROLOG

```
This will allow to include our typical FHiCL files also _after_ including this one.
Otherwise, no FHiCL configuration inside a `PROLOG` will be accepted after this.
(FHiCL rule is: first all the `PROLOG`s, then everything outside it)